src/components/button/Button.js

```diff
@@ -12,9 +12,8 @@ import PropTypes from 'prop-types';
  * @param {typography} props capitalize, uppercase, lowercase, only
  */
 const Button = (props) => {
-  const { children, type, size, shape, typography, disabled, fullWidth, id, onClick } = props;
+  const { children, type, size, shape, typography, disabled, fullWidth, id, onClick, className } = props;
```

Comment:
Is it possible not to pass className down? The reason for this is when people start creating their own classes and pass it down if we want to change the base class or extend it, it will break someone's implementation.

The react team were also saying they have seen that practice to make it hard later on because if you want change the base class that will inevitbly break someone's implementation